modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/src/main/resources/content/Language.properties

```diff
@@ -7,6 +7,7 @@ diagram-file=Diagram File
 diagram-mapping=Diagram Mapping
 diagram-pin=Diagram Pin
 diagram-settings=Diagram Settings
+no-diagram-loaded-yet=No diagram loaded yet
```

Comment:
I am not a native English speaker, but this seems off. What I do know is that this does not match out convention for labels in UI. A label can be:
1. A title, with each work capitalized. For example, `Diagram Settings`.
2. A sentence, with first word capitalized and dot on the end. For example, `This diagram shows shopping items.`

Check with your product folks what to put here. I'd recommend against label, rather put a spinner here. See `ClayLoadingIndicator`, commerce is using it a lot.

Reply:
I'm adding the dot to the sentence, thank you!
I've put that component because that was the design for this case. Thanks anyway!

Reply:
Fair enough, product folks have the last word.